packages/core/src/pipeline/config/triggers/cdevents/cdevents.trigger.ts

```diff
@@ -0,0 +1,19 @@
+import { CDEventsTrigger } from './CDEventsTrigger';
+import { ArtifactTypePatterns } from '../../../../artifact';
+import { Registry } from '../../../../registry';
+
+Registry.pipeline.registerTrigger({
+  component: CDEventsTrigger,
+  description: 'Executes the pipeline when a CDEvents webhook is received.',
+  excludedArtifactTypePatterns: [ArtifactTypePatterns.JENKINS_FILE],
```

Comment:
Is there a reason for excluding Jenkins file artifacts?

Reply:
nothing specifically, but I just followed the Webhook trigger type as this is similar to CDEvents